OsmAnd/src/net/osmand/plus/mapcontextmenu/controllers/RenderedObjectMenuController.java

```diff
@@ -26,12 +27,20 @@ public class RenderedObjectMenuController extends MenuController {
 
 	private RenderedObject renderedObject;
 
+	private String nameStr = null;
+	private MapPoiTypes mapPoiTypes = null;
+	private MapPoiTypes.PoiTranslator poiTranslator = null;
+
 	public RenderedObjectMenuController(@NonNull MapActivity mapActivity,
 	                                    @NonNull PointDescription pointDescription,
 	                                    @NonNull RenderedObject renderedObject) {
 		super(new RenderedObjectMenuBuilder(mapActivity, renderedObject), pointDescription, mapActivity);
 		builder.setShowNearestWiki(true);
 		setRenderedObject(renderedObject);
+		if (this.getMapActivity() != null) {
```

Comment:
we get nonnull mapActivity in constructor, do we need to use geter & check here?

Also app or poiTypes could be stored in local variable to avoid copy pasted calls